BOOTSTRAP.adoc

```diff
@@ -97,6 +97,9 @@ There are five steps to renaming a primitive:
 It is desirable for bootstraps to be easily repeatable, so you should commit
 changes after step 4.
 
+To remove a primitive, start with a working build of the compiler eg `./configure && make`
+then and only then remove the primitive and repeat steps 4 and 5 above.
+
```

Comment:
I think this is a useful addition, but I find the wording “repeat steps 4 and 5” a bit unclear. Repeating implies that I already performed them, so should I perform them first, them `./configure && make`, then perform them again?

Also I wonder whether it is necessary to run a full build with `make`. Trying to infer the steps to remove a primitive from the steps to rename one:

1. Remove uses of the primitive, but not the primitive itself. Then ensure the system still works:
   ```
   make coreall
   ```
2. Then, and only then, remove the primitive, and run
   ```
   make coreall
   make bootstrap
   ```

That would seem clearer to me. But I’m not sure whether it really works, or whether it’s minimal.

Reply:
That is a better wording, I didn't need to remove uses of the primitive since it was unused in the first place, but in future it is the more likely situation. I've updated the instructions to make it more like your suggestion.